webhook_server/libs/github_api.py

```
@@ -647,6 +652,31 @@ def _repo_data_from_config(self, repository_config: dict[str, Any]) -> None:
+        merged_labels_config = {**global_labels_config, **repo_labels_config}
+
+        # enabled-labels: if not set, all labels enabled (None means all enabled)
+        self.enabled_labels: set[str] | None = None
+        if "enabled-labels" in merged_labels_config:
+            enabled_set = set(merged_labels_config["enabled-labels"])
+            # Log warning for invalid categories
+            invalid = enabled_set - CONFIGURABLE_LABEL_CATEGORIES
+            if invalid:
+                self.logger.warning(
+                    f"{self.log_prefix} Invalid label categories in enabled-labels config: {invalid}. "
+                    f"Valid categories: {CONFIGURABLE_LABEL_CATEGORIES}"
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)